lib/addons/src/types.ts

```diff
@@ -253,10 +253,20 @@ export interface BaseMeta<ComponentType> {
 }
 
 export interface BaseStory<Args, StoryFnReturnType> {
-  (args: Args, context: StoryContext): StoryFnReturnType;
-
+  (args: Args | StoryContext, context?: StoryContext): StoryFnReturnType;
```

Comment:
```suggestion
  (context: StoryContext): StoryFnReturnType;
  (args: Args, context: StoryContext): StoryFnReturnType;
```

Reply:
I didn't know that was possible, nice suggestion!